modules/tinymce/src/core/main/ts/api/html/Node.ts

```diff
@@ -491,6 +491,12 @@ class Node {
             return false;
           }
 
+          // Keep anchor in empty block
+          const isAnchor = !node.attr('href') && (node.attr('id') || node.attr('name'));
```

Comment:
This also has the same issues as my previous comment and looks to duplicate the logic on the next lines, as mentioned by the comment `Keep bookmark nodes and name attribute like <a name="1"></a>`. As such we should either merge that logic into that block, or remove the parts that's trying to keep it atm.

- [ ] Rename variable
- [ ] Clean up duplicate logic